Mod1/Lessons/Week5/Scope.md

```diff
@@ -0,0 +1,232 @@
+# Scope and Access Modifiers
+
+## Learning Goals
+* Identify how **scope** affects where variables can be accessed
+* Have a reference to look at when **scope** errors show up
+* Understand the difference between the **public** and **private** access modifiers
+
+# Scope
+
+## Intro
+Scope is what you have access to and where you have access to it.
```

Comment:
I pulled this intro from the turing backend scope lesson, but looking back on it now I wonder if it's too wordy. I think in general I write too many words, let me know if you have suggestions for things to cut!

Reply:
I don't think this is too wordy!  But, I do think we might want to get students mental juices flowing before we dive right in.  Maybe some kind of warm-up exercise that gets them thinking about when _they_ have experienced scope issues.  It could be as simple as 'here's a screenshot of a time when I didn't have access to the variable I wanted'... have you encountered this before? What did you do to fix the issue?